syncd/NotificationHandler.cpp

```diff
@@ -93,7 +111,9 @@ void NotificationHandler::onPortStateChange(
 {
     SWSS_LOG_ENTER();
 
-    auto s = sai_serialize_port_oper_status_ntf(count, data);
+    auto ntfdata = Workaround::convertPortOperStatusNotification(count, data, m_apiVersion);
+
+    auto s = sai_serialize_port_oper_status_ntf((uint32_t)ntfdata.size(), ntfdata.data());
```

Comment:
@kcudnik I think you need to parse the new field `sai_port_error_status_t` in sai_serialize_port_oper_status_ntf() otherwise this new field will not be posted to redis DB and OA will not see this field in the notification